service/controller/v15/resource/cloudformation/current.go

```diff
@@ -51,17 +52,6 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 
 	r.logger.LogCtx(ctx, "level", "debug", "message", "found the guest cluster main stack in the AWS API")
 
-	// In case the current guest cluster is already being updated, we cancel the
-	// reconciliation until the current update is done in order to reduce
-	// unnecessary friction.
-	if stackStatus == cloudformation.ResourceStatusUpdateInProgress {
```

Comment:
In general `ResourceStatusUpdateInProgress == StackStatusUpdateInProgress`. I just added `StackStatusUpdateInProgress` it here: https://github.com/giantswarm/aws-operator/pull/1099/files#diff-6d2b44d389deb52257928547dee86ac0R60